src/Datadog.Trace.ClrProfiler.Managed/Extensions/SpanContextPropagatorExtensions.cs

```diff
@@ -0,0 +1,111 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using Datadog.Trace.ClrProfiler.Emit;
+using Datadog.Trace.Logging;
+
+namespace Datadog.Trace.ClrProfiler.Extensions
+{
+    internal static class SpanContextPropagatorExtensions
+    {
+        private const NumberStyles NumberStyles = System.Globalization.NumberStyles.Integer;
+
+        private static readonly CultureInfo InvariantCulture = CultureInfo.InvariantCulture;
+        private static readonly Vendors.Serilog.ILogger Log = DatadogLogging.GetLogger(typeof(SpanContextPropagatorExtensions));
+
+        public static void InjectHttpHeadersWithReflection(this SpanContextPropagator spanContextPropagator, SpanContext context, object headers)
```

Comment:
Seems weird to make these extensions on `SpanContextPropagator` if the `this` argument is not used.

Reply:
Interesting...I was trying to follow the convention in `SpanContextPropagator.cs` where `Inject` and `Extract` are made on instances of `SpanContextPropagator`, but the instance is never used at all. Is that correct, or am I missing something about the original `SpanContextPropagator`?

Reply:
Hah, I guess not. Looks like `SpanContextPropagator` could have be a static class. I'm guessing I probably didn't do that (made it a singleton instead) to make it easier to mock if we wanted to.

Reply:
What if I move it to the `Helpers` directory, rename to `SpanContextPropagatorHelpers`, and make them normal static methods instead of extension methods? Does that sound good?

Reply:
I refactored it to just be a helper class instead of a class with extension methods in 6f12b0b3eeab9d07ce3b864a39b7254f94981ea0

Reply:
Makes sense. You wouldn't have the `SpanContext` instance to make extraction an extension, anyway. Thanks!